index.js

```diff
@@ -36,6 +36,16 @@ const VM_OPTIONS = {
 };
 const vm = new VM(VM_OPTIONS);
 
+//--- Built-ins
+
+vm.run(
```

Comment:
We could then potentially make all of these functions immutable via Object.defineProperty with a `vm.run()` line that iterates through the properties of `sandbox`.